app/controllers/api/base_controller.rb

```diff
@@ -31,8 +31,6 @@ class BaseController < ActionController::API
     before_action :ensure_pagination, :only => :index
     after_action :log_api_response
 
-    respond_to :json
-
```

Comment:
Wow, that was easy.  Was it too easy?  Are you sure it's not used?  🤯

Reply:
From what I read, the purpose of this method is to setup the metadata so `respond_with` works. But we don't use `respond_with`...

Where I got my information:

https://github.com/heartcombo/responders/blob/956cd874a997083d350951fde4c6cc4944f64e80/lib/action_controller/respond_with.rb#L17-L38

Reply:
It hurts my soul that there is no gemspec deletion for repsonders needed here.  I guess the dependency wasn't moved over here when the repo was extracted.  Oh well.

LGTM though.